### DISCUSSION TITLE
Add Support to Repeater to hide Repeater item after Add-Button Press

### DISCUSSION THREAD

**Discussion by amueller82** [formkit/formkit#1190] [Ideas]
**Idea:**
Enhance the Repeater so that the current Repeater item can optionally be hidden after the Add-Button has been pressed.

This would enable situations where a Repeater is used to input multiple items to a list - so that the Repeater-Item appears to be reused by the User to submit/edit data. This would also clean-up the UI - in cases where the user only whats to have one visual form to be filled out.

**Example:** Adding Attendees to an Event

Currently we are using Javascript to manipulate the DOM and change the Class of the Repeater Item to hide the repeater item with the respective class. (e.g. d-none in bootstrap)

https://formkit.link/1817c549c6fe2f8bfc1364a890d823e5

**Comment by justin-schroeder** [comment]
Would each repeater item be hidden entirely, or sort of "minimized"?

**Reply by amueller82** [reply]
Sort of minimized properly would be most natural to the repeater item and would also ease up the "Edit-Item" Case when an entry needs to be edited afterwards. 

A collapsable Accordion is what comes to my mind first.. -> https://getbootstrap.com/docs/4.0/components/collapse/#accordion-example
